src/Containers/Doctors/Doctors.js

```diff
@@ -37,8 +38,12 @@ const Doctors = () => {
 
   return(
     <>
-      <h2>Doctors here</h2>
-      <section>{allDoctors}</section>
+      <h2 className='dr-sub-title'>Doctors here</h2>
+      <section>{allDoctors}
+      <Link to='/'>
+      <button className='home-button'>Home</button>
+      </Link>
+      </section>
```

Comment:
Having a home button is great! We might want to move it to the nav bar in the future